docs/how-to-docs/cert-manager.md

```diff
@@ -22,37 +50,37 @@ Migrating from the former [kube-cert-manager](https://github.com/PalmStoneGames/
 
 #### **As a developer I want to retrieve an internal certificate**
 
-As stated above the cert-manager can also handle internal certificates i.e. those signing the internal ACP Certificate Authority _(this is self signed btw)_. At the moment you might be using [cfssl-sidekick](https://github.com/UKHomeOffice/cfssl-sidekick) to perform this, but this can be completely replaced.
+As stated above the cert-manager can also handle internal certificates i.e. those signed by the internal ACP Certificate Authority _(this is self signed btw)_. At the moment you might be using [cfssl-sidekick](https://github.com/UKHomeOffice/cfssl-sidekick) to perform this, but this can be completely replaced.
 
 ```YAML
-apiVersion: certmanager.k8s.io/v1alpha1
+apiVersion: cert-manager.io/v1alpha2
 kind: Certificate
 metadata:
   name: tls
 spec:
   secretName: tls
   issuerRef:
-    name: platform-tls
+    name: platform-ca
     kind: ClusterIssuer
   commonName: site.svc.project.cluster.local
   dnsNames:
   - localhost
   - 127.0.0.1
 ```
 
-This would create a kubernetes secret named `tls` in your namespace with the signed certificate. An interesting thing to note here although this is using the ClusterIssuer platform-ca created by the ACP team, there is nothing stopping a project from creating a local Issuer for the own project. So for example.
+This would create a kubernetes secret named `tls` in your namespace with the signed certificate. An interesting thing to note here is that although this is using the ClusterIssuer platform-ca created by the ACP team, there is nothing stopping a project from creating a local Issuer for the own project. So for example.
```

Comment:
` for the own project.` => ` for their own project.`

Reply:
k